packages/website/src/components/editor/useSandboxServices.ts

```diff
@@ -145,7 +145,10 @@ export const useSandboxServices = (
         model.dispose();
       }
     };
-  }, [props.ts, colorMode, props.jsx, onLoaded]);
+    // colorMode can't be reactive here, this is going to cause crash while switching dark/light mode
```

Comment:
Could you elaborate here on why?

Reply:
not sure what else you want me to write here, i pushed small fix for jsx, (chaning jsx option was causing crash to)

Reply:
I mean, _why_ would providing `colorMode` cause a crash?

Reply:
hmm, how to put it in proper sentence...

> in this context `useEffect` is used as constructor/destructor, constructor uses singletor to generate code and destructor is triggered always causing code to not be correctly initialized.
> This forces external lib to recreate editor when we want to update theme or one of options.

i'm bad at explaining stuff xd

Reply:
Ah I think I got it! How about...

```suggestion
    // colorMode can't be reactive here because we don't want to force a recreation
    // of the editor whenever options or theme changes
```